fendermint/app/src/app.rs

```diff
@@ -187,6 +191,7 @@ where
             state_store: Arc::new(state_store),
             multi_engine: Arc::new(MultiEngine::new(1)),
             builtin_actors_bundle: config.builtin_actors_bundle,
+            custom_actors_bundle: config.custom_actors_bundle,
```

Comment:
At some point we could follow up with a PR that allows a list here, so users can load their own bundles, all identified by name.